src/ingredientHydrator.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace GRUB;
+
+class IngredientHydrator {
+    /**
+     * @var array of instantiated ingredient entity objects
+     */
+    public $results;
+
+    /**
+     * IngredientHydrator constructor.
+     *
+     * @param Db $db
+     *
+     * @result populates results property with ingredients' names
+     */
+    public function __construct(\PDO $db)
+    {
+        $query = $db->prepare("SELECT `name` FROM `ingredients`;");
+        $query->setFetchMode(\PDO::FETCH_CLASS, 'IngredientEntity');
```

Comment:
This is not working as it should, this returns an array of arrays rather than an array of your IngredientEntities.

Reply:
```suggestion
        $query->setFetchMode(\PDO::FETCH_CLASS, 'GRUB\IngredientEntity');
```

Reply:
Fixed